### PR TITLE
Add landing page for my blog

### PR DESCRIPTION
**Pull request by teohm**
I added a new landing page.

Take a look

### CODE REVIEW

**Comment by teohm** [comment]
@KentNg201 @cloudsoh Please take a look 👀 🙇

File: index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+<body>
```
Comment (cloudsoh) [inline index.html]: Remember to insert indentation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+<h1>Welcome to my blog</h1>
+
+<p>TODO</p>
```
Comment (teohm) [inline index.html]: This line is just a placeholder, we will add content later

**Review by cloudsoh — APPROVED** [review]

**Review by kentng201 — APPROVED** [review]
🚢 👍

**Review by kentng201 — APPROVED** [review]
OK what

**Pull request merged**